ui/admin/app/components/form/user/add-accounts/index.js

```diff
@@ -30,12 +33,14 @@ export default class FormUserAddAccountsComponent extends Component {
    * Accounts not already added to the user.
    * @type {[AccountModel]}
    */
-  @computed('args.{accounts.[],model.account_ids.[]}')
   get filteredAccounts() {
     // Get IDs for accounts already added to the current user
     const alreadyAddedAccountIDs = this.args.model.account_ids;
+    // TODO: Remove the addAccount ability check during Phase 2
     const notAddedAccounts = this.args.accounts.filter(
-      ({ id }) => !alreadyAddedAccountIDs.includes(id)
+      (account) =>
+        this.can.can('addAccount account', account) &&
```

Comment:
Did we want to actually filter out the ldap accounts or just disable them to prevent users from trying to add them? Should we still let them see it?

Reply:
Great question! Had a discussion with my pod sync and we agreed to exclude ldap accounts from the [add-accounts](https://boundary-ui-git-icu-9226-disable-all-actions-f-d5c52f-hashicorp.vercel.app/scopes/global/users/u_1r8fkwwiqj/add-accounts) route since that feels like the better ux because a user expects to see a list of accounts that could be added.

Reply:
@joshuaogle We would also appreciate your thoughts on this.

Reply:
Alrighty, change was applied and this is the new [preview](https://boundary-ui-git-icu-9226-disable-all-actions-f-d5c52f-hashicorp.vercel.app/scopes/global/users/u_uab8iflhfu/add-accounts).  Users can now see disabled ldap accounts in the add-accounts route.